### PR TITLE
Extract modal to separate page, Add materials info

### PR DESCRIPTION
**Pull request by junhyukee**
Sorry! Was my first PR and did not create a separate branch. I did so, and am submitting a new PR.

Hello, this pull request addresses issues #89 and #102. Please let me know if anything can be improved, and I'll get right on it! If I should use a different page name, instead of product-details, let me know and I'll quickly change that as well. 

* [X] Add a new page to the site
* [X] Add the content from the modal to the new page
* [X] Swap out modal links to for links to the new page
* [X] Remove any unused code and/or dependencies from the modal
* [X] Add T-Shirt Material and Fit content

### CODE REVIEW

File: src/components/ProductDetails/ProductDetails.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import styled, { injectGlobal } from 'react-emotion';
```
Comment (jlengstorf) [inline src/components/ProductDetails/ProductDetails.js]: I don't think `injectGlobal` is needed in this file, is it?

File: src/components/ProductDetails/ProductDetails.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+          <li>Material: 100% cotton</li>
+          <li>Fit:</li>
+            <ul>
+              <li>All sizes: regular/retail fit</li>
+            </ul>
+        </UnorderedList>
+        <Subheading>Size Chart</Subheading>
+
+        <UnitSelector unit={this.state.units} setUnits={this.changeUnits} />
+        <SizeChartTable unit={this.state.units} />
```
Comment (jlengstorf) [inline src/components/ProductDetails/ProductDetails.js]: Looking at this, it seems like the size chart would be better at the top of this list.

File: src/components/Store/Store.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import styled from 'react-emotion';
+import Link from 'gatsby-link';
```
Comment (jlengstorf) [inline src/components/Store/Store.js]: Since this is a Gatsby V2 site, we now pull in links from:

```
import { Link } from 'gatsby';
```

**Comment by junhyukee** [comment]
Thanks for the guidance, @jlengstorf! I implemented the changes, could you take a look? I took the liberty of doing some typography changes to help distinguish sections, hope that was okay!

* [X] Remove injectGlobal
* [X] Link import
    - import link from gatsby instead of gatsby-link
* [X] Ordering change:
    - Care, Materials&Fit, Size Chart to Size Chart, Materials & Fit, Care
* [X] Little bits of typography change
    - Made subheading font-size a bit bigger to emphasize a section change
    - Made nested UnorderedList use disc instead of circles

**Comment by jlengstorf** [comment]
I can’t figure out why this build is failing, because the build works for me locally. I'm wondering if Netlify has locked up on this particular branch. I'm going to merge this 😅 and see if the master branch builds. If not, I'll dig deeper and figure out what's going on. Either way, it's nothing to do with the code in this PR.

Thanks again!

**Pull request merged**